tests/workflow_contracts/makefile_tooling_test.py

```
@@ -0,0 +1,219 @@
+
+    completed = _run_make("nixie", env)
+
+    assert completed.returncode == 0, completed.stderr
+    invocations = _read_invocations(log_path)
+    repository_tmp = str(REPOSITORY_ROOT / ".tmp")
+    uv_cache = str(REPOSITORY_ROOT / ".uv-cache")
+    uv_tools = str(REPOSITORY_ROOT / ".uv-tools")
+    assert invocations[:2] == [
+        f"bun|{repository_tmp}|{uv_cache}|{uv_tools}|install --frozen-lockfile",
+        f"bun|{repository_tmp}|{uv_cache}|{uv_tools}|"
+        "scripts/install-mermaid-browser.mjs",
```

Ruff (0.16.0)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)